.gitignore

```diff
@@ -15,8 +15,6 @@ coverage/
 .mailmap
 component.json
 debug/local/
-Gemfile.lock
 yarn.lock
 *.log
 .eslintcache
-.jekyll-cache
```

Comment:
I've moved all the Jekyll related files that should be ignored to their own `.gitignore` in the `docs` directory.